website/pages/en/querying/graphql-api.mdx

```diff
@@ -376,11 +389,11 @@ Graph Node implements [specification-based](https://spec.graphql.org/October2021
 
 ## Schema
 
-The schema of your data source--that is, the entity types, values, and relationships that are available to query--are defined through the [GraphQL Interface Definition Langauge (IDL)](https://facebook.github.io/graphql/draft/#sec-Type-System).
+The schema of your `data source`, i.e. the entity types, values, and relationships that are available to query, are defined through the [GraphQL Interface Definition Langauge (IDL)](https://facebook.github.io/graphql/draft/#sec-Type-System).
```

Comment:
Not sure why "data source" is inline code here – looks like the field in the manifest is spelled `dataSources`. I would keep it as regular text I think, or maybe change it to `dataSource` but even that feels a bit wrong since the actual field is plural.